src/WorkflowManager/WorkflowManager/Monai.Deploy.WorkflowManager.csproj

```diff
@@ -60,6 +60,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\Shared\Configuration\Monai.Deploy.WorkflowManager.Configuration.csproj" />
     <ProjectReference Include="..\..\Shared\Shared\Monai.Deploy.WorkflowManager.Shared.csproj" />
+    <ProjectReference Include="..\..\TaskManager\Plug-ins\AideClinicalReview\Monai.Deploy.WorkflowManager.TaskManager.AideClinicalReview.csproj" />

```

Comment:
Don't think this should be here

Reply:
We need this as we're validating against required clinical review keys in the `args` param and the `Keys` constants are in this project.

Reply:
Hmm I think its still better that we keep the projects separate and we duplicate these keys somewhere within the workflow manager.